flatdata-go/flatdata/byte_array_handle.go

```diff
@@ -0,0 +1,45 @@
+/*
+Copyright (c) 2017 HERE Europe B.V.
+See the LICENSE file in the root of this project for license details.
+*/
+
+package flatdata
+
+import (
+	"fmt"
+	"io"
+)
+
+// byteArrayHandle implements ResourceHandle interface with reading date from []byte for internal usage
```

Comment:
Is `Handle` a usual name for pointer-like structures? In C++ implementation we use `MemoryDescriptor`. Probably, we want to unify the names later.

Reply:
We decided about naming with @imagovrn some time ago. It should be similar to other implementations, for sure, but looks like we need to improve it. Unfortunately, I don't know other implementations very well...